### PR TITLE
Prevent mobile table td collapsing issue

### PR DESCRIPTION
**Pull request by sg00dwin**
Fixes https://github.com/openshift/origin-web-console/issues/1128

screen shot 2017-01-17 at 4 44 41 pm

### CODE REVIEW

**Comment by spadgett** [comment]
In other places, we used a non-breaking space to make sure the table cell had height. For instance,

https://github.com/spadgett/origin-web-console/blob/695c74dc0fac848444c79c555fb689c4d8431973/app/views/browse/routes.html#L82

If we have a different solution, we should probably change those to be consistent.

File: app/styles/_tables.less [hunk 1]
```diff
@@ -123,6 +123,7 @@
            /* Behave like a row */
           border: none;
           border-bottom: 1px solid lighten(@table-border-color, 5%);
+          min-height: 38px; // prevent empty td collapsing
```
Comment (spadgett) [inline app/styles/_tables.less]: This seems like a magic number. Is there any existing variable that makes sense for min-height?

**Comment by spadgett** [comment]
cc @rhamilto

**Comment by sg00dwin** [comment]
Updated PR, switched to using non-breaking space to maintain td height

**Comment by spadgett** [comment]
I'm good with the CSS fix. Just saying we should remove the `nbsp` in other places if it's no longer needed to clean up our markup.

**Comment by sg00dwin** [comment]
Updated pr to use min-height calculated. 

Note: related issue pertaining to switching out other `&nbsp;` used to give height, will be corrected in other pr.

**Comment by spadgett** [comment]
[merge]

**Pull request merged**